packages/react-native/React/Base/macOS/RCTUIKit.m

```diff
@@ -454,7 +454,12 @@ - (void)setTransform:(CGAffineTransform)transform
 
 - (NSView *)hitTest:(NSPoint)point
 {
-  return [self hitTest:NSPointToCGPoint(point) withEvent:nil];
+// [macOS
+  // IMPORTANT point is passed in super coordinates by OSX, but expected to be passed in local coordinates
+  NSView *superview = [self superview];
+  NSPoint pointInSelf = superview != nil ? [self convertPoint:point fromView:superview] : point;
+  return [self hitTest:pointInSelf withEvent:nil];
+// macOS]
```

Comment:
We don't need macOS diff tags in RCTUIKit, because the whole file is a macOS only file
```suggestion
  // IMPORTANT point is passed in super coordinates by OSX, but expected to be passed in local coordinates
  NSView *superview = [self superview];
  NSPoint pointInSelf = superview != nil ? [self convertPoint:point fromView:superview] : point;
  return [self hitTest:pointInSelf withEvent:nil];
```

Reply:
ah. I was wondering if I could omit it. I figured I'll include it just in case and remove it if you say so.